.github/workflows/ci.yml

```
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  qa:
+    name: QA
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toxenv:
+          - checkqa
+          - readme
```

You use this matrix value later but didn't declare it:
```suggestion
          - readme
        python-version:
        - 3.8
```

---

I'd rather remove the `python-version`, forgot to do that.